public/media_engagement/worldbank_plr_sms_traffic.js

```diff
@@ -45,9 +45,10 @@ firebase.auth().onAuthStateChanged(function (user) {
 
 const TIMEFRAME_WEEK = 7;
 const TIMEFRAME_MONTH = 30;
-var chartTimeUnit = "1day";
+var chartTimeUnit = "10min";
 var isYLimitReceivedManuallySet = false;
 var isYLimitSentManuallySet = false;
+extend_x_max_by_hours = (date) => date.setHours(date.getHours() + 24);
```

Comment:
This modifies the date and returns it. Please do one or the other

Reply:
Could you also change the function to name to something like 'add_one_day_to_date'?

 - By saying extend by 'hours' it sounds like users should be able to specify the number of hours, which they can't. 
 - This function's signature and implementation know nothing of x_max, so I think it's unnecessary to include that in the name. The function becomes much more general purpose with that name change.

Reply:
@as2388 please take a look on this changes that you have requested

Reply:
looks better.
 - Is day of type number or Date? The function name suggests it should be a date but the variable suggests a number - please update the function name or variable name appropriately
 - Does JS provide getDays and setDays functions that allow you to setDays(getDays() +1)? I think that would be more readable

Reply:
Day is of type Date, so the function name is correct. I'll just change the variable name.

Reply:
Yes it does let me change to days. Thank you @as2388 this has helped a great deal.